MetaMask/metamask-extension
# Add reveal to export private key

## Explanation

Changes
- Add hold modal step prior to revealing private key
- add tests
- migrate to use design system components

## Screenshots/Screencaps

### Before
[before-hold-in-export-private.webm](https://user-images.githubusercontent.com/96463427/225314378-2c718a57-7339-42ce-8642-b455b234eb56.webm)

### After
[after-hold-in-export-private.webm](https://user-images.githubusercontent.com/96463427/225314385-c51645b9-b4df-4239-bbe4-7762cea60c44.webm)

## Manual Testing Steps

1.  Click on account details
2.  Click on `Export private key`
3.  Enter password and click `Confirm`
4.  Click and hold on `Hold to reveal SRP`

## Pre-merge author checklist

- [ x] I've clearly explained:
  - [x ] What problem this PR is solving
  - [ x] How this problem was solved
  - [ x] How reviewers can test my changes
- [ x] Sufficient automated test coverage has been added
